vector/src/main/java/im/vector/app/features/home/HomeActivity.kt

```diff
@@ -559,10 +546,6 @@ class HomeActivity :
                         putExtra(MvRx.KEY_ARG, args)
                     }
         }
-
-        private const val MATRIX_TO_CUSTOM_SCHEME_URL_BASE = "element://"
```

Comment:
"element://" was there at first place (in app side)

Reply:
Indeed, I observed you already moved this scheme from the service to the activity in a previous commit...